test/svid/x509svid/test_x509_svid.py

```diff
@@ -0,0 +1,389 @@
+import pytest
+from cryptography.hazmat.primitives import serialization
+from cryptography.x509 import Certificate
+
+from pyspiffe.spiffe_id.spiffe_id import SpiffeId
+from pyspiffe.svid.exceptions import (
+    InvalidLeafCertificateError,
+    InvalidIntermediateCertificateError,
+    ParseCertificateError,
+    ParsePrivateKeyError,
+    LoadCertificateError,
+    LoadPrivateKeyError,
+)
+from pyspiffe.svid.x509_svid import X509Svid
+from cryptography.hazmat.primitives.asymmetric import ec, rsa
+
+_TEST_CERTS_PATH = 'test/svid/x509svid/certs/{}'
+
+
+def test_parse_raw_chain_and_ec_key():
+    chain_bytes = read_bytes(_TEST_CERTS_PATH.format('1-chain.der'))
```

Comment:
I see that `read_bytes` is a helper function defined here for the tests. Is there any reason that prevents from setting the path internally in the function? That way `_TEST_CERTS_PATH.format('file_name')` occurs only once.

Reply:
Done.